_docs_operate/configuration.md

```diff
@@ -110,6 +112,56 @@ You can validate the config using our [schema file](https://raw.githubuserconten
 
 The debug flag configures if the Connector is set to **production** or **debug** mode. Defaults to `false`. Can also be configured using the environment variable `DEBUG`.
 
+### enforceCertificatePinning `available since version 6.5.0` {#enforceCertificatePinning}
+
+The `enforceCertificatePinning` flag configures if the Connector should enforce certificate pinning. Defaults to `false`.
```

Comment:
The `enforceCertificatePinning` flag configures ~if~whether the Connector should enforce certificate pinning.